src/test/kotlin/uk/gov/dluhc/registercheckerapi/job/RegisterCheckMonitoringJobIntegrationTest.kt

```diff
@@ -18,37 +30,35 @@ internal class RegisterCheckMonitoringJobIntegrationTest : IntegrationTest() {
     @Autowired
     protected lateinit var registerCheckMonitoringJob: RegisterCheckMonitoringJob
 
+    @Mock
+    private lateinit var emailService: EmailService
```

Comment:
Mocking at the EmailService level doesn't feel right for a test that is describing itself as an integration test (though probably there should also be non-integration tests for the RegisterCheckMonitoringService job which is probably my fault sorry!)

Reply:
Good job getting the integration tests working! 😄